Wflow/src/sediment/sediment_transport/river_transport.jl

```diff
@@ -181,17 +181,17 @@ function SedimentRiverTransportParameters(
     if any(abs.(river_fractions .- 1.0) .> 1e-3)
         error("Particle fractions in the river bed must sum to 1")
     end
-    lens = lens_input_parameter(config, "clay__d50_diameter")
+    lens = lens_input_parameter(config, "clay_mean_diameter")
```

Comment:
Just a very quick drive-by comment, shouldn't it have double underscores as well? Same holds for other changes as well
```suggestion
    lens = lens_input_parameter(config, "clay__mean_diameter")
```

Reply:
You are very right! Thanks for spotting this